CHANGELOG.md

```diff
@@ -8,8 +8,9 @@
 
 ##### Bug Fixes
 
-* None.  
-
+* Fix "can't modify frozen string" errors when pods are integrated using the `branch` option
```

Comment:
nit: please add two empty spaces at the end of this line for markdown formatting.